apps/guardian/src/pages/Actions/DecryptionActions.tsx

```diff
@@ -129,13 +169,26 @@ export function DecryptionActions() {
             {step === TallyStep.TallyError && <DecryptionError>Could not read the election tally</DecryptionError>}
             {step === TallyStep.GenerateDecryptionShare && <GenerateDecryptionShare />}
             {step === TallyStep.AwaitPeerShares && (
-                <AwaitPeers predicate={(g) => g.hasDecryptionShare}>
-                    Waiting for peers to register their shares of the decryption
+                <AwaitPeers
+                    predicate={(g) => g.hasDecryptionShare}
+                    note={
+                        <>
+                            The window for registering decryption shares closes in: <br />
+                            <b>
+                                <Countdown to={electionConfig.decryptionDeadline} />
+                            </b>
+                        </>
+                    }
+                >
+                    Waiting for peers to register their shares of the decryption.
```

Comment:
Shouldn't the terminology here match the registration terminology.

As I read it it will say
```
Generate decryption proof
```

and here it's talking about shares of decryption.

Reply:
At this step it's waiting for all guardians to register their decryption shares, or for the decryption deadline to run out.

The steps in this component are listed in the order they will be executed by the guardian, so the step prior to this "waiting" step is "generate decryption share" and the step after is "generate decryption proof".

Otherwise I don't understand what you're trying to say 😅